packages/block/src/header.ts

```diff
@@ -66,12 +66,12 @@ export class BlockHeader {
   }
 
   /**
-   * EIP-4399: Post-PoS merge, `mixHash` supplanted as `random`
+   * EIP-4399: After merge to PoS, `mixHash` supplanted as `prevRandao`
    */
-  get random() {
+  get prevRandao() {
```

Comment:
What's this for a strange name? 🤔

Reply:
Some explanation here: https://twitter.com/TimBeiko/status/1494768704143298560